src/gltf-component-mappings.js

```diff
@@ -423,4 +423,26 @@ AFRAME.GLTFModelPlus.registerComponent("audio-settings", "audio-settings", (el,
   el.sceneEl.systems["hubs-systems"].audioSettingsSystem.updateAudioSettings(componentData);
 });
 
-AFRAME.GLTFModelPlus.registerComponent("video-texture-target", "video-texture-target");
+AFRAME.GLTFModelPlus.registerComponent(
+  "video-texture-target",
+  "video-texture-target",
+  (el, componentName, componentData, _components, indexToEntityMap) => {
+    const { targetBaseColorMap, targetEmissiveMap, src } = componentData;
+
+    const target = indexToEntityMap[src];
+    if (!target) {
+      console.warn(
+        `Error inflating gltf component "video-texture-target": Couldn't find target entity with index ${src}`
+      );
+      return;
```

Comment:
Nope good catch. I think I would have caught that when testing for regressions in video avatars which I planned to do, but yeah this is obviously incorrect. This might be the right behavior if `target` was specified in the gltf but not found. Because in that case there was some sort of error in the asset.